### ISSUE TITLE
Error: Type 'Boolean' has no call signatures.

### ISSUE THREAD

**Issue by L04DB4L4NC3R** [oakserver/oak#62]
## What was done

A file called **app.ts** was created with the following code:

```typescript
import { Application } from "https://deno.land/x/oak/mod.ts";

const app = new Application();

app.use(ctx => {
  ctx.response.body = "Hello World!";
});

console.log("Listening.....");
await app.listen({ port: 8000 });
```

Subsequently, the following command was executed:

```sh
deno --allow-net app.ts
```

---

## Expected output

Expected deno to compile the code successfully and see the following line in stdout:

```
Listening.....
```

---

## Actual Output

Compilation failed with the following error:

```
Compile file:///home/angad/Desktop/covidcomm/covidcomm/app.ts
error TS2349: This expression is not callable.
  Type 'Boolean' has no call signatures.

► https://deno.land/x/oak/send.ts:62:36

62     return (await Deno.stat(path)).isFile();
                                      ~~~~~~

error TS2349: This expression is not callable.
  Type 'Boolean' has no call signatures.

► https://deno.land/x/oak/send.ts:139:15

139     if (stats.isDirectory()) {
                  ~~~~~~~~~~~


Found 2 errors.
```

Error screenshot:
![image](https://user-images.githubusercontent.com/30529572/80010390-fa4ec700-84e7-11ea-97fd-a7f112add18b.png)

## Environment

| Package | Version | 
|:--:|:--:|
| deno | 0.41.0 |
| v8 | 8.2.308 |
| typescript | 3.8.3 |

![image](https://user-images.githubusercontent.com/30529572/80010510-22d6c100-84e8-11ea-9fa1-25064950629d.png)

**Comment by macko911** [comment]
This is fixed in #58 once merged.

For now you can import the fixed version this way
`import { Application } from "https://denopkg.com/Soremwar/oak@v0.41/mod.ts";`

**Comment by kitsonk** [comment]
This is fixed in master now.  Will release a tagged version soon.
